.github/workflows/test-gnofract4d.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python: [3.8, 3.9]
+        python: [3.8, 3.9, "3.10"]
```

Comment:
Does this need the quotes to work or can it be done without? If it needs them, we suggest converting the other values here to use them too, otherwise we'd drop the quotes around 3.10

Reply:
Without the quotes it tries to use Python 3.1. Hadn't realised that before pushing here. Updated to use quotes around all Python versions.